README.md

```diff
@@ -130,6 +130,9 @@ RUN wget https://github.com/sendgrid/aws-env/releases/download/1.0.0/aws-env -O
 CMD aws-env --region us-east-2 /my-app
 ```
 
+### Use as a Docker image
+Starting with version `1.2.1`, we publish a Docker image that contains the `aws-env` binary.
```

Comment:
should we say `v1.2.1` here, for consistency?

Reply:
Yes